src/app/EventManagement.cpp

```diff
@@ -584,6 +595,12 @@ static bool IsInterestedEventPaths(EventLoadOutContext * eventLoadOutContext, co
     {
         return false;
     }
+
+    if (event.mFabricIndex.HasValue() && eventLoadOutContext->mFabricIndex != event.mFabricIndex.Value())
```

Comment:
So in particular, the way things are right now any events generated by a PASE session before it's picked up a fabric association will be visible to all consumers.

Maybe that's not a problem in practice because you can't add ACLS via a PASE session before it picks up a fabric?  But we would need to pretty carefully check the ordering of things in AddNOC....

On the other hand, if we allowed event.mFabricIndex to have the "no fabric" value (0).... I guess a later PASE session would be able to see the events, which is not desirable?

Maybe the right answer is that fabric-scoped events emitted not in the context of a fabric should just be dropped completely?  Worth checking with @mlepage-google and @tcarmelveilleux whether they have thoughts on this.  But we can also adjust it as a followup.

Reply:
Thanks I took a note to think about this.

Reply:
https://github.com/project-chip/connectedhomeip/issues/14366